packages/plugin-core/src/_extension.ts

```diff
@@ -907,173 +894,17 @@ async function showWelcomeOrWhatsNew({
 
   // Show lapsed users (users who have installed Dendron but haven't initialied
   // a workspace) a reminder prompt to re-engage them.
-  if (shouldDisplayLapsedUserMsg()) {
-    await showLapsedUserMessage(assetUri);
+  if (StartupUtils.shouldDisplayLapsedUserMsg()) {
+    await StartupUtils.showLapsedUserMessage(assetUri);
   }
 
   // Show inactive users (users who were active on first week but have not used lookup in 2 weeks)
   // a reminder prompt to re-engage them.
-  if (shouldDisplayInactiveUserSurvey()) {
-    await showInactiveUserMessage();
+  if (StartupUtils.shouldDisplayInactiveUserSurvey()) {
+    await StartupUtils.showInactiveUserMessage();
```

Comment:
nitpicking/up to your decision, but how about moving the if parts inside the `StartupUtils` too? So this would be just `StartupUtils.showMessageIfInactiveUser` or something like that. Then these functions could be private to `StartupUtils`.

Reply:
That sounds like a cleaner way to do this. 👍🏼